### ISSUE TITLE
Tutorial hard to follow

### ISSUE THREAD

**Issue by dkarigithu** [JaspervRijbroek/raddish#193]
Hi Jasper,

Loving your tutorial, want to give it a whirl but I keep hitting 404 error, component not found. Any chance you could setup a repo with the complete code. Really annoying trying to get up and running for something I'm building.

Regards.

**Comment by JaspervRijbroek** [comment]
@dkarigithu Hi David,

I am very happy to hear that you are using raddish.
I will create some examples for you which are located in this repo: [JaspervRijbroek/raddish-examples](https://github.com/JaspervRijbroek/raddish-examples)

Although I started working on these this morning, so it might take some time for them to be complete.

If things are still unclear please let me know.

Thanks a lot,

Jasper

**Comment by JaspervRijbroek** [comment]
@dkarigithu I also have another project I am working on you can use as an example [aura-cp](https://github.com/JaspervRijbroek/aura-cp).

Kind Regards,

Jasper

**Comment by dkarigithu** [comment]
Thanks @JaspervRijbroek, your a genius with this framework, been looking for a Multi-threaded node.js framework, this is absolutely epic. Fact that you use Ember as well, kills it. Keep up the great work will keep following you and your future projects.
